app/helpers/menus_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module MenusHelper
+  def food_items_for_select(food_items)
+    food_items.map { |item| [item.name, item.id, data: { price: item.price }] }
+  end
+
+  def user_role_status(current_user)
+    return t('menus_messages.new.helpers.admin') if current_user.admin?
+
+    return t('menus_messages.new.helpers.customer') unless current_user
```

Comment:
hmm, I'm not sure if this method is working at all.
The result is gonna be `Customer` if `current_user == nil`, and `nil` when `current_user is not an admin`.
Anyway, as far as I see you can remove `return` and `unless blah-blah` in last line. 
Real talk, think about it.